ThemeBuilder: Add a note about saving metadata (#1833)
DevExpress/devextreme-documentation: concepts/60 Themes and Styles/08 ThemeBuilder/01 Get Started/3 Create a New Theme.md

3. Select a color scheme and change basic appearance settings.

> 4. If the appearance on the preview meets you requirements, click **Export** to [export the theme](/concepts/60%20Themes%20and%20Styles/08%20ThemeBuilder/40%20Export%20the%20Resulting%20Theme.md '/Documentation/Guide/Themes_and_Styles/ThemeBuilder/#Export_the_Resulting_Theme') to a stylesheet. We also recommend that you export your theme's metadata. You will need it to update your custom theme if you decide to migrate to a future DevExtreme version.       

Reviewer: - Phrases like "meets you requirements" or "does not meet your requirements" do not explain anything. Why would they not meet your requirements? In this case, does this mean you can use custom themes? If so, why not just state that you can also use custom themes?
- "You will need it to update" Why do you neet to use the future tense here? "You need [it] to update" (you should specify what 'it' is)

```suggestion
4. If you want to use a custom theme, click **Export** to [export the theme](/concepts/60%20Themes%20and%20Styles/08%20ThemeBuilder/40%20Export%20the%20Resulting%20Theme.md '/Documentation/Guide/Themes_and_Styles/ThemeBuilder/#Export_the_Resulting_Theme') to a stylesheet. We also recommend that you export your theme's metadata. You need the stylesheet to update your custom theme when you migrate to a newer DevExtreme version.       
```

Author: This article describes the process of creating a new custom theme. It's up to the user to decide how it should look like. As for the future tense, the described events (that is, updating the custom theme) may or may not happen in the future. Now the metadata isn't required, because there's no need to update the custom theme since it has been created seconds ago. If a user decide to migrate to a new version, he will need this metadata. The same note answers your comment below.

Reviewer: - How does this justify vague phrases like **meets you(r) requirements** and **are not enough**? Can you quantify 'your requirements' or 'not enough'? Why or how does it not meet your requirements? How do you measure this? What is not enough? What would be enough? 
- I do not see suitable justification for the future tense either

Reviewer: Does it mean you need more customization options? More color choices?

> If the basic appearance settings are not enough, click **Advanced Settings**, and you will navigate to the main view for [additional customization](/concepts/60%20Themes%20and%20Styles/08%20ThemeBuilder/10%20Customize%20the%20Theme.md '/Documentation/Guide/Themes_and_Styles/ThemeBuilder/#Customize_the_Theme').

Reviewer: - "are not enough" This is the same as the comment regarding "meets you requirements". Can you quantify 'enough'? What is not enough? 
- I'm not sure why you need the future tense here either
